io/writeCometsLayout.m

```diff
@@ -370,8 +370,51 @@ function writeCometsLayout( input, filedir, filename, includeParams, writeModels
         end
     end
 end
+fprintf(fileID,'\t//\n');
+
 
-fprintf(fileID,'\t//\n//\n');
+%% 2g Periodic media
+% This block describes what and where should be the periodic media.
+% There are two settings, global and detailed. 
+% Global:
+% metIndex  functionName   amplitude period  phase offset
+%
+% Detailed:
+% metIndex functionName    rowIndex  colIndex amplitude period phase offset
+
+if layout.periodic_media_mode == "global"   
+    % Global mode
+    fprintf(fileID,'\tperiodic_media\tglobal\n');
+    % Iterate over array and print if amplitude is > 0
+    for i=1:length(layout.global_periodic_media)
+        params = layout.global_periodic_media{i};

```

Comment:
minor edit, but can we rename this to something like "gpm" because "params" is already a term used on a broader scope?

Reply:
Done